hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/HddsPolicyProvider.java

```diff
@@ -36,8 +36,14 @@
 @Unstable
 public final class HddsPolicyProvider extends PolicyProvider {
 
-  private static final Supplier<HddsPolicyProvider> SUPPLIER =
-      MemoizedSupplier.valueOf(HddsPolicyProvider::new);
+  private static final Supplier<HddsPolicyProvider> SUPPLIER = MemoizedSupplier.valueOf(HddsPolicyProvider::new);
```

Comment:
Can we avoid reformat-only changes?

Reply:
I understand.  It's just that there was nothing wrong with the original.  There will inevitably be some classes with long name where these need to be wrapped anyway.

This one is wrapped badly and I agree should be changed:

```java
  private static final Logger LOG = LoggerFactory
      .getLogger(XceiverServerRatis.class);
```